packages/web/src/scss/components/SkipLink/README.md

```diff
@@ -0,0 +1,32 @@
+# SkipLink
+
+The `SkipLink` component is used to provide a way for users to skip directly to the e.g. main content of a page,
+improving accessibility for keyboard and screen reader users.
+
+## Usage
+
+```html
+<body>
+  <nav aria-label="Navigation with skip links">
+    <a href="#main-content" class="SkipLinkItem"> Skip to main content </a>
+  </nav>
+  <!-- Other navigation elements -->
+  <main id="main-content">
+    <!-- Main content -->
+  </main>
+</body>
+```
+
+⚠️ Ensure that the `SkipLink` is placed before the section you want to skip to.
+
+### 👍 When to Use
+
+- Provide a way for sighted keyboard or screen reader users to bypass repetitive navigation
```

Comment:
It serves to skip not only repetitive navigation. You can add it to complex tables too, or to ASCII art :D